.github/workflows/release-apps.yaml

```
@@ -0,0 +1,95 @@
+        working-directory: python/apps/${{ matrix.app }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Extract Python version from .python-version
+        id: extract-python
+        run: |
+          PYTHON_VERSION=$(head -n 1 .python-version)
+          echo "version=$PYTHON_VERSION" >> "$GITHUB_OUTPUT"
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)